qcengine/compute.py

```diff
@@ -63,26 +97,17 @@ def compute(input_data: Union[Dict[str, Any], 'ResultInput'],
     # Run the program
     with compute_wrapper(capture_output=False) as metadata:
 
-        output_data = input_data.copy()  # Initial in case of error handling
-        try:
-            output_data = get_program(program).compute(input_data, config)
-        except KeyError as e:
-            output_data = FailedOperation(
-                input_data=output_data.dict(),
-                success=False,
-                error=ComputeError(
-                    error_type='program_error',
-                    error_message="QCEngine Call Error:\nProgram {} not understood."
-                    "\nError Message: {}".format(program, str(e))))
+        output_data = input_data.copy()  # lgtm [py/multiple-definition]
+        output_data = executor.compute(input_data, config)
 
     return handle_output_metadata(output_data, metadata, raise_error=raise_error, return_dict=return_dict)
 
 
-def compute_procedure(input_data: Dict[str, Any],
+def compute_procedure(input_data: Union[Dict[str, Any], 'BaseModel'],
```

Comment:
Are we going to start using Type Hint Annotations everywhere? And because I cant remember, doesn't this require the `from __future__ import annotations` invocation to use in 3.6?

Reply:
This is only required for forward annotations which have no elected to do, hence the string fields. We are still pretty far from being able to run mypy.